pythonAPIClient/client.py

```diff
@@ -79,7 +83,92 @@ def update_camera(self, camID, others):
     def camera_by_id(self, cameraID):
         pass
 
-    # TODO: search a camera
-    # replace others with desired field names
-    def search_camera(self, others):
-        pass
\ No newline at end of file
+    def search_camera(self, lat=None, lng=None, radius=None, type=None, source=None,
+                      country=None, state=None, city=None, resolution_width=None,
```

Comment:
we should change lat, lng to latitude and longitude

Reply:
done